builtin/credential/okta/path_login.go

```diff
@@ -172,6 +182,43 @@ func (b *backend) pathLoginRenew(ctx context.Context, req *logical.Request, d *f
 	return resp, nil
 }
 
+func pathVerify(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `verify/(?P<nonce>.+)`,
+		Fields: map[string]*framework.FieldSchema{
+			"nonce": {
+				Type: framework.TypeString,
+				Description: `Nonce provided during a login request to
+retrieve the number verification challenge for the matching request.`,
+			},
+		},
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ReadOperation: &framework.PathOperation{
+				Callback:                    b.pathVerify,
+				ForwardPerformanceSecondary: true,
+				ForwardPerformanceStandby:   true,
```

Comment:
Just gave this a test using a perf standby. I think we'll need to either forward both the login/verify requests or not forward both of them. Right now, only the verify request is forwarded, which means that logins initiated on perf standby/secondary aren't able to complete the verify step (the nonce isn't cached on the instance the request is forwarded to).

I'd guess that it's probably fine to not forward this. The client would complete the full login/verify on the Vault instance it started the login with.

Reply:
You're right. The login request should be handled locally on perf secondaries, and the login handler does not seem to have any storage writes so it can be processed by perf standbys.